src/Components/MixedType/MixedType.scss

```diff
@@ -0,0 +1,14 @@
+@import '../../styles/Colors.scss';
+.mixed {
+    width:100%;
+    input[type=radio] {
+        margin: 0px;
+    }
+    .optionName {
+        margin-left:10px;
+    }
+    input[type=radio]:before {
+        border-radius:20% !important;
```

Comment:
all radio buttons needs to be super unique across the application, so keep this border-radius globally